usaspending_api/etl/management/commands/load_executive_compensation.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.db import connections
+
+from usaspending_api.etl.broker_etl_helpers import PhonyCursor
+from usaspending_api.etl.executive_compensation_etl import load_executive_compensation
+
+logger = logging.getLogger('console')
+exception_logger = logging.getLogger("exceptions")
+
+
+class Command(BaseCommand):
+    """
+    This command loads the executive compensation data from the broker using
+    either a list of DUNS or all duns present in the data store
+    """
+    help = "Loads a set of DUNS's executive compensation data from the configured data broker database"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-d',
+            '--duns',
+            dest="duns",
+            nargs='+',
+            type=int,
+            help="DUNS to load subawards for"
+        )
+
+        parser.add_argument(
```

Comment:
Ah yes, this is just a leftover from a bit of copy and pasting, I will fix that and clarify.